### PR TITLE
Added the new comixed-web submodule to replace the UX [#328]

### PR DESCRIPTION
**Pull request by mcpierce**
## Status
**READY**

## Migrations
NO

## Description
Created a new submodule, comixed-web, that provides an Angular app based on Angular Material for the user interface. Add the initial features to allow a user to login and logout, a confirmation dialog and an alert service, and a simple feature set based on NgRx.

### CODE REVIEW

File: comixed-rest-api/src/test/java/org/comixedproject/controller/user/UserControllerTest.java [hunk 1]
```diff
@@ -138,7 +139,10 @@ public void testSetUserPropertyForInvalidUser() throws ComiXedUserException {
 
   @Test
   public void testGetCurrentUserWhenNotAuthented() throws ComiXedUserException {
```
Comment (BRUCELLA2) [inline comixed-rest-api/src/test/java/org/comixedproject/controller/user/UserControllerTest.java]: it's a detail but it seems to me that there is a mistake in the name of the method (but I can be wrong, my English is not very good).

File: comixed-web/pom.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        <version>1.7.5</version>
+        <executions>
+          <execution>
+            <id>Install node and yarn</id>
+            <goals>
+              <goal>install-node-and-yarn</goal>
+            </goals>
+            <phase>initialize</phase>
+            <configuration>
+              <nodeVersion>v14.7.0</nodeVersion>
```
Comment (BRUCELLA2) [inline comixed-web/pom.xml]: lastest version for node is v14.15.0. Do we need to update it ?

**Review by BRUCELLA2 — COMMENTED** [review]
It's ok for me, juste 2 questions / comments.

File: comixed-rest-api/src/test/java/org/comixedproject/controller/user/UserControllerTest.java (hunk 1 above)
Comment (mcpierce) [reply comixed-rest-api/src/test/java/org/comixedproject/controller/user/UserControllerTest.java]: Good catch. Fixed it.

File: comixed-web/pom.xml (hunk 2 above)
Comment (mcpierce) [reply comixed-web/pom.xml]: I'm good with updating things to the latest. Done!

**Review by BRUCELLA2 — APPROVED** [review]

**Pull request merged**